src/main/java/server/Client.java

```diff
@@ -21,10 +21,14 @@ public Client(Socket clientSocket) throws IOException {
     public PrintWriter getWriter() { return writer; }
 
     public String chunkStream() {
-        char[] charBuffer = new char[100000];
+        char[] charBuffer = new char[1000];
         try {
-            int bytes_read = reader.read(charBuffer);
-            return new String(charBuffer, 0, bytes_read);
+            String input = "";
+            while(reader.ready()) {
+                int bytes_read = reader.read(charBuffer);
+                input += new String(charBuffer, 0, bytes_read);
+            }
+            return input;
         } catch (IOException | StringIndexOutOfBoundsException error) {
             ServerLogger.serverLogger.log(Level.WARNING, "Error: " + error);
             return null;
```

Comment:
What happens in this scenario? Are we returning a 500?

Reply:
We are now!